API/Articles.js

```diff
@@ -0,0 +1,17 @@
+import axios from "axios";
+
+const apiClient = axios.create({
+    baseURL: "https://northcoders-backend-project.onrender.com/api/",
+    timeout: 1000,
```

Comment:
this might be problematic only because render which you are using for your backend spins down after I think 15minutes of activity.